.eslintrc.js

```diff
@@ -37,6 +37,8 @@ module.exports = {
         "@typescript-eslint/type-annotation-spacing": "off",
         "@typescript-eslint/no-use-before-define": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
+        "no-use-before-define": "off",
```

Comment:
Not in this diff, but it seems `@typescript-eslint/no-use-before-define` is written twice.

Reply:
Removed the duplicate entry